.github/workflows/release.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
   test:
     name: run tests and linters
-    uses: kubewarden/github-actions/.github/workflows/reusable-test-policy-go.yml@v1
+    uses: kubewarden/github-actions/.github/workflows/reusable-test-policy-go.yml@v2
```

Comment:
```suggestion
    uses: kubewarden/github-actions/.github/workflows/reusable-test-policy-go.yml@v2.0.2
```

Reply:
Done in https://github.com/kubewarden/ingress-policy/pull/26/commits/5b9fb15a59a794fe1ed4914a4fe7e29f9f655ede!